templates/catalog/product.html

```diff
@@ -83,13 +83,13 @@
                 </td>
 
                 <td class="option-td">
-                  {% include 'catalog/count_input.html' %}
+                  {% include 'catalog/count_input.html' with option=option only %}
```

Comment:
it is not necessary there
```suggestion
                  {% include 'catalog/count_input.html' %}
```

Reply:
@artemiy312 , `with var1=var1 var2=var2 only` is never necessary. We use it all over the templates as an additional isolation level now